### ISSUE TITLE
[Feature Request] Batch Process - merge short lines 

### ISSUE THREAD

**Issue by Reinbowsaur** [SubtitleEdit/subtitleedit#3867]
Any plans to add this feature of joining short lines to the batch screen? I can't seem to achieve this currently, and it would be very useful.

Thank you for the program its great as it is now too.

**Comment by niksedk** [comment]
@Reinbowsaur: could you try latest beta?
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.11/SubtitleEditBeta.zip

**Comment by Reinbowsaur** [comment]
> @Reinbowsaur: could you try latest beta?
> https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.11/SubtitleEditBeta.zip

Hey I've played around with it abit, it looks great. 

Some convert options inside the batch convert, do not allow custom settings, in my case "Merge short-lines" does not show an settings button when selected inside batch convert, as do a few other options not allow custom settings. I would want to do a batch convert using "MergeShortLines" with a few custom parameters such as in the below image, I would want to disable "Only merge continuation lines" for my partiular batch of files. But can't since I can't adjust the default settings used by the batch convert.
![image](https://user-images.githubusercontent.com/51115162/70401062-91edd480-1a75-11ea-8a27-98cb91d1024c.png)

I did try to get around this by editing my profile settings and adding the line 
> "onlyMergeContinuationLines":"False"
it did not work, I assume it does not work like this/or will not work at all/I have guessed the wrong name for the parameter I am trying to change the default to false.

ps. This is probably normal, I also wanted more than the default 43 Singlemaxlinelength, so I changed it to 300 in the general settings, that produced an error when opening up merge short lines, since merge short lines usually only allows you to change 43, to 200.

I thank for you for your time and work on this.
